float64 target_x
float64 target_y
float64 target_theata
---
float64 goal_x
float64 goal_y
float64 goal_theta
---
float64 current_x
float64 current_y
float64 current_theta